server/src/codeAction.ts

```diff
@@ -46,5 +46,5 @@ function toCopName(diagnostic: Diagnostic): string {
 }
 
 function toCorrectable(diagnostic: Diagnostic): boolean {
-  return (diagnostic.data as any).correctable;
+  return diagnostic.data == null ? false : (diagnostic.data as any).correctable;
```

Comment:
Since diagnostic.data is unknown type, using optional chaining operator might be more suitable in this case.

```suggestion
  return (diagnostic.data as any)?.correctable;
```